connection/connection.go

```diff
@@ -19,3 +21,11 @@ type AWSKinesisSource struct {
 	AWSSecretAccessKey string `json:"awsSecretAccessKey,omitempty"`
 	AWSSessionToken    string `json:"awsSessionToken,omitempty"`
 }
+
+// KafkaSource is the configuration struct to leverage when defining a Kafka resource
+// TODO: add tls config portion
+type KafkaSource struct {
```

Comment:
Let's focus the initial pass on Kinesis.

Reply:
Yes, just wanted to add color that there would be additional source types later. We were making `Source` previously into `AWSKinesis` stuff only.